sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client_connection_async.py

```diff
@@ -143,14 +144,6 @@ def __init__(
         # Keeps the latest response headers from server.
         self.last_response_headers = None
 
-        if consistency_level == documents.ConsistencyLevel.Session:
```

Comment:
this comment is for line 141:
    if consistency_level is not None:
            self.default_headers[http_constants.HttpHeaders.ConsistencyLevel] = consistency_level

the above statements have been removed from sync client implementation, why not using similar logic here as well?

Reply:
because we need to store the level of consistency the user is passing somewhere in order to do the checks later to see if starting a session is necessary